x-pack/test/apm_api_integration/tests/service_overview/instances_main_statistics.spec.ts

```diff
@@ -418,6 +418,16 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             expect(javaStats).to.be(undefined);
           });
 
+          it('does not return metrics', () => {
+            const goAStats = body.currentPeriod.find(
+              (stat) => stat.serviceNodeName === 'go-instance-a'
+            );
+
+            expect(goAStats).to.not.be(undefined);
+            expect(goAStats?.memoryUsage).to.be(undefined);
+            expect(goAStats?.cpuUsage).to.be(undefined);
```

Comment:
Does this test reproduce the original error? In other words: will this test fail without the changes to the scripts (wrapping in should clause)

Reply:
no, I added additional tests to catch the original error https://github.com/elastic/kibana/pull/145348/commits/530546bc3af80489f728c283829fdc6e24494136